driver-kafka/src/main/java/io/openmessaging/benchmark/driver/kafka/KafkaBenchmarkConsumer.java

```diff
@@ -57,12 +87,6 @@ public KafkaBenchmarkConsumer(KafkaConsumer<byte[], byte[]> consumer, ConsumerCa
                 }
             }
         });
+        return future;
```

Comment:
I don't see where this future is getting completed.

Reply:
Doh...I forgot about that.